### PR TITLE
Expand `Ranked_score`

### PR DESCRIPTION
**Pull request by WindowsMeosu**
## Self-check

- [ 

- [x]  The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
waiting for a person who speaks french fluently to review [this.](https://github.com/WindowsMeosu/osu-wiki/blob/patch-1/wiki/Gameplay/Score/Ranked_score/fr.md)
waiting for a person who speaks Indonesian flunently to review [this.](https://github.com/WindowsMeosu/osu-wiki/blob/patch-1/wiki/Gameplay/Score/Ranked_score/id.md)

### CODE REVIEW

**Comment by WindowsMeosu** [comment]
requested files changed

**Comment by cl8n** [comment]
was this closed on accident? seems like good info for this page

**Comment by WindowsMeosu** [comment]
> was this closed on accident? seems like good info for this page

I can reopen this if you or anyone wants and the reason I closed was because I lost hope in this since [checks ](https://github.com/ppy/osu-wiki/pull/7774/checks)kept failing and just decided it wasn't worth it.

**Comment by Walavouchey** [comment]
no need to lose hope over the checks, if nothing else they can be fixed by a maintainer before merging anyway

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (wowcake) [inline wiki/Gameplay/Score/Ranked_score/id.md:9]:
```suggestion
**Skor ranked** (*ranked score*) adalah kumpulan dari jumlah seluruh skor terbaik dari seluruh tingkat-tingkat kesulitan beatmap Ranked yang dimainkan secara *online*.
```

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (wowcake) [inline wiki/Gameplay/Score/Ranked_score/id.md:13]:
```suggestion
Sebelum ppv1 dan ppv2, peringkat ditentukan oleh jumlah skor ranked yang dimiliki oleh pemain. Pemain peringkat #1 akan sering berganti karena betapa kecilnya osu! pada dulunya dan juga kekurangan map. Skor ranked telah sepenuhnya digantikan oleh ppv1 pada 24 Juli 2012, yang kemudian digantikan oleh ppv2 pada 27 Januari 2014.
```

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (wowcake) [inline wiki/Gameplay/Score/Ranked_score/id.md:15]:
```suggestion
## Approved Beatmap
```

Comment (wowcake) [reply wiki/Gameplay/Score/Ranked_score/id.md:15]: no need to translate this one

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (wowcake) [inline wiki/Gameplay/Score/Ranked_score/id.md:17]:
```suggestion
*Approved Beatmap* adalah kategori untuk beatmap yang memberikan skor yang sangat banyak (misalnya Marathon). Approved Beatmap tidak memberikan skor ranked, namun memberikan Total Score dan pp. Juga, Loved map adalah map yang tidak memberikan pp namun memberikan Ranked dan Total score.
```

**Review by wowcake — CHANGES_REQUESTED** [review]
other than those reviews, looks good enough

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (WindowsMeosu) [reply wiki/Gameplay/Score/Ranked_score/id.md:13]: commited

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (WindowsMeosu) [reply wiki/Gameplay/Score/Ranked_score/id.md:15]: guess I'll commit anyways~

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (WindowsMeosu) [reply wiki/Gameplay/Score/Ranked_score/id.md]: commited!

File: wiki/Gameplay/Score/Ranked_score/id.md
Comment (WindowsMeosu) [reply wiki/Gameplay/Score/Ranked_score/id.md:17]: commited~

thanks again

**Comment by WindowsMeosu** [comment]
appears I ain't getting nothing done and my laziness is starting to prevent me from working on this so I'm just gonna close since I'm probably not gonna get anything done atm.

**Comment by Walavouchey** [comment]
if you want, you can leave the pr open so that we can fix any remaining issues ourselves before merging. the only reason nothing has happened is that maintainers haven't taken a look at this yet.

**Comment by WindowsMeosu** [comment]
gonna rework this :3

**Pull request closed without merging**
